requirements-test.txt

```diff
@@ -2,5 +2,6 @@
 # compatible with yours. If so, install PyTorch with the correct CUDA version
 # as instructed on https://pytorch.org/get-started/locally/
 torch
-pytest==3.2.5
-pytest-cov==2.5.1
+pytest
+pytest-cov
+packaging
```

Comment:
Please pin the major version to avoid breaking changes. E.g., `pytest>=3.2,<4`.

Reply:
The pytest and pytest-cov packages are changed. I think `packaging` need not be fixed because it uses a year-based version, and the version string comparison logic should not likely be changed.